tests/renderStatsCard.test.js

```diff
@@ -344,7 +344,7 @@ describe("Test renderStatsCard", () => {
       document.querySelector(
         'g[transform="translate(0, 100)"]>.stagger>.stat.bold',
       ).textContent,
-    ).toMatchInlineSnapshot(`"参与项目数:"`);
+    ).toMatchInlineSnapshot(`"参与项目数 (last year):"`);
```

Comment:
The parenthesized `（star）`, `（commit）`, `（PR）`, and `（issue）` above are annotating the Chinese terms `星`, `提交`, `拉取请求`, and `问题`.
`(last year)` breaks parallelism both syntactically (half-width vs full-width parentheses) and semantically (should be "过去一年" rather than "last year").

I understand it's an i18n problem. Maybe worth tracking it somewhere? @anuraghazra

Reply:
@liyishuai, you are 100% correct. When I added that code, I didn't have time to implement the i18n translations; feel free to create an issue and implement a PR. I'm happy to review it. I think you should wait until https://github.com/anuraghazra/github-readme-stats/issues/2053#issuecomment-1407432409 is implemented since this will change the translation pipeline a bit.